mako/__main__.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env python3
+
+"""
+entry point for directly running mako as templating engine.
+
+that way, mako can directly be used as an interpreter via shebang:
+#!/usr/bin/env -S python3 -m mako
```

Comment:
mako-render is already installed as a command on its own.  why can't the command be "#!/usr/bin/env mako-render -s" ?

Reply:
sure, it can be. I just wanted to introduce the possibility of executing the `mako` module by `python3 -m mako`, but it's not necessary since we do have the `mako-render` entrypoint.

`env` needs to be called with `-S` though so it splits the command line arguments - otherwise `mako-render` would get only one argument string.